packages/design-tokens/src/tokens/functional/components/video-player/colors.js

```diff
@@ -27,8 +27,8 @@ module.exports = {
       },
       controls: {
         bgColor: {
-          value: 'linear-gradient(var(--base-color-scale-transparent), #01040966)',
-          dark: 'linear-gradient(var(--base-color-scale-transparent), #01040966)',
+          value: 'rgba(0,0,0,0.75)',
+          dark: 'rgba(0,0,0,0.75)',
```

Comment:
Can these be changed to hex format please. That's the preferred format for color values in Primitives now.

```suggestion
          value: '#000000bf',
          dark: '#000000bf',
```